frontend/src/pages/Home/components/DataModels/Layouters/utils.js

```diff
@@ -0,0 +1,88 @@
+import { Position, internalsSymbol } from 'reactflow';
+import ResponseError from "../../../../../ResponseError";
+
+
+
+function getHandleCoordsByPosition(node, handlePosition) {
+    // all handles are from type source, that's why we use handleBounds.source here
+    const handle = node[internalsSymbol].handleBounds.source.find(
+        (h) => h.position === handlePosition
+    );
+
+    let offsetX = handle.width / 2;
+    let offsetY = handle.height / 2;
+
+    // this is a tiny detail to make the markerEnd of an edge visible.
+    // The handle position that gets calculated has the origin top-left, so depending which side we are using, we add a little offset
+    // when the handlePosition is Position.Right for example, we need to add an offset as big as the handle itself in order to get the correct position
+    switch (handlePosition) {
+        case Position.Left:
+            offsetX = 0;
+            break;
+        case Position.Right:
+            offsetX = handle.width;
+            break;
+        case Position.Top:
+            offsetY = 0;
+            break;
+        case Position.Bottom:
+            offsetY = handle.height;
+            break;
+        default :
+            throw new ResponseError("Position should be Left, Right, Top or Bottom", Response.error())
```

Comment:
```suggestion
        default :
            throw new Error("Position should be Left, Right, Top or Bottom", Response.error())
```

Pas la bonne sémantique